packages/fileeditor-extension/src/index.ts

```diff
@@ -316,11 +316,14 @@ function activate(app: JupyterLab, restorer: ILayoutRestorer, editorServices: IE
             break;
           }
         }
-      } else {
+        selected = blocks.length > 0;
```

Comment:
This misses the case for a line that is in a markdown file that contains code blocks, but is not in any of the code blocks. I don't know how common that is, but it was surprising to me as I tested this.

For instance, a markdown file with this will run the first line:
```
    print("Hello world")

```
while this will not:

```
    print("Hello world")

    ```
    print('I am a spoiler')
    ```
```